modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -31,6 +42,22 @@ def find_bgs_claim!(claim_id:)
 
           raise
         end
+
+        def attempt_submit(bgs_claim)
+          response = bgs_service.benefit_claims.update_bnft_claim(claim: bgs_claim)
+
+          if !response[:bnft_claim_dto].nil?
+            bgs_claim[:status] = ClaimsApi::EvidenceWaiverSubmission::UPDATED
```

Comment:
isn't the `bgs_claim` here the claim that we received from BGS and not the one within our table? 

On that note, we also need to create the `ClaimsApi::EvidenceWaiverSubmission` somewhere before attempting to set the status here.